atum/src/main/scala/za/co/absa/atum/core/ControlType.scala

```diff
@@ -0,0 +1,26 @@
+package za.co.absa.atum.core
+
+class ControlType(val value: String)
```

Comment:
Do I understand it correctly that you've used `class` instead of `sealed trait` here so that you can instantiate `ControlType` with an arbitrary string?

Reply:
Exactly